test/socket.jl

```diff
@@ -239,19 +239,21 @@ begin
     close(sock)
 end
 
-# Local-machine multicast
-
-let
-    function create_socket()
-        s = UDPSocket()
-        bind(s, ip"0.0.0.0", 2000, reuseaddr = true, enable_broadcast = true)
-        s
-    end
-    a, b, c = [create_socket() for i = 1:3]
-    @sync begin
-        recvs = [@async @test bytestring(recv(s)) == "hello" for s in (a, b)]
-        send(c, ip"255.255.255.255", 2000, "hello")
-        map(wait, recvs)
+# Local-machine broadcast
+# (Mac OS X's loopback interface doesn't support broadcasts)
+if @osx ? false : true
```

Comment:
No I wasn't, but echoing @sbromberger comment on #14040 it doesn't seem like a good idea to generate active network traffic as part of a test, which is the case when using `255.255.255.255`. Also, if somebody were to run a somewhat restricted firewall on Mac, this test would fail.